docs/DeveloperGuide.md

```diff
@@ -69,7 +69,7 @@ Eslint and its accompaning modules can be installed through NPM, so do ensure th
 ### Building and running RepoSense from code
 
 1. Execute the following command on the OS terminal inside the project directory. <br/>
-Usage: `gradlew run -Dargs="([-config CONFIG_FOLDER] | -repos REPO_PATH_OR_URL... | -view REPORT_FOLDER) [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]"` <br/>
+Usage: `gradlew run -Dargs="([-config CONFIG_FOLDER] | -repos REPO_PATH_OR_URL... | -view REPORT_FOLDER) [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...] [--ignore-standalone-config]"` <br/>
```

Comment:
Can add the alias in the DeveloperGuide as well

Reply:
Hmmm, how about `-isac, --ignore-standalone-config` (followed the format in link above)?

Reply:
do you mean `[--ignore-standalone-config, -isac]` ?

Reply:
Hmm, maybe `-isac/--ignore-standalone-config` instead?

Reply:
OK, that's a good idea.